homeassistant/components/smartthings/sensor.py

```diff
@@ -229,6 +229,9 @@
     ],
     Capability.three_axis: [],
     Capability.tv_channel: [Map(Attribute.tv_channel, "Tv Channel", None, None)],
+    Capability.tv_channel: [
```

Comment:
You're overwriting the key for `tv_channel` -- add it to the value side, which is an array similar to:

```python
Capability.tv_channel: [
    Map(Attribute.tv_channel, "Tv Channel", None, None),
    Map(Attribute.tv_channel_name, "Tv Channel Name", None, None),
]
```

Reply:
Thanks for the hint.
Updated accordingly.